packages/host/app/components/ai-assistant/new-session.gts

```diff
@@ -1,6 +1,5 @@
-import { fn } from '@ember/helper';
+import { TemplateOnlyComponent } from '@ember/component/template-only';
```

Comment:
import `type` { TemplateOnlyComponent }. A small thing but I've come across bizarre lint errors when forgetting to import this as `type` in other parts of the repo